api/src/main/java/org/pmiops/workbench/google/DirectoryService.java

```diff
@@ -5,4 +5,6 @@
  */
 public interface DirectoryService {
   public boolean isUsernameTaken(String username);
+  public void createUser(String givenName, String familyName, String username, String password);
```

Comment:
It may be worth noting that calling this method creates a GSuite account, which actually costs us money every month. :)

Reply:
I think the invitation key satisfies this.